config/settings/prod.py

```
@@ -17,3 +17,48 @@
+    },
+}
+
+LOGGING["handlers"] = {
+    "console": {
+        "class": "logging.StreamHandler",
+        "formatter": "medium",
+        "filters": ["correlation_id"],
+    },
+    "file": {
+        "class": "logging.handlers.TimedRotatingFileHandler",
+        "filename": BASE_DIR / "logs" / "prod.log",
```

Ruff (0.15.6)

[error] `BASE_DIR` may be undefined, or defined from star imports

(F405)